spec/models/news/importer_spec.rb

```diff
@@ -0,0 +1,116 @@
+require 'rails_helper'
+
+RSpec.describe News::Importer do
+  subject(:instance) { described_class.new json_data }
+
+  let(:json_data) { file_fixture 'news/govuk_stories.json' }
+
+  describe '.new' do
+    it { is_expected.to be_instance_of described_class }
+  end
+
+  describe '#stories' do
+    subject { instance.stories }
+
+    it { is_expected.to have_attributes length: 3 }
+  end
+
+  describe '#import!' do
+    it { expect { instance.import! }.to change(News::Item, :count).by(3) }
+    it { expect(instance.import!).to be 3 }
```

Comment:
Instance seems to return the number of stories,
and not those that are imported.